coalib/results/result_actions/ShowPatchAction.py

```diff
@@ -81,7 +81,7 @@ def apply(self,
               file_diff_dict,
               colored: bool=True):
         '''
-        Print a diff of the patch that would be applied.
+        Show patch.
```

Comment:
there are more actions lying around in result_actions, can you change all of them?